coral-schema/src/main/java/com/linkedin/coral/schema/avro/RelToAvroSchemaConverter.java

```diff
@@ -453,8 +453,14 @@ public RexNode visitCall(RexCall rexCall) {
 
     @Override
     public RexNode visitOver(RexOver rexOver) {
-      // TODO: implement this method
-      return super.visitOver(rexOver);
+      /**
+       * For aggregates over window, derive th expected field's schema using RexOver's return type alone.
+       * Example RexOver:
+       *  ROW_NUMBER() OVER (PARTITION BY colA, colB ORDER BY colC DESC)
+       */
+      appendField(rexOver.getType(), false,
```

Comment:
Do we need to get the nullability via `SchemaUtilities.isFieldNullable(rexOver, inputSchema);`?

Reply:
Sounds good, thanks for explanation.